compiler/src/iree/compiler/Dialect/VM/Conversion/VMToEmitC/EmitCBuilders.h

```diff
@@ -18,9 +18,17 @@ namespace mlir {
 namespace iree_compiler {
 namespace emitc_builders {
 
+typedef struct StructField {
```

Comment:
nit: no typedef needed in c++, just 
```suggestion
struct StructField {
  std::string type;
  std::string name;
};
```